pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -109,12 +109,24 @@ type ExecCommand struct {
 	LabeledCommand `json:",inline"`
 
 	// The actual command-line string
+	//
+	// Special variables that can be used:
+	//
+	//  - `$PROJECTS_ROOT`: A path path where projects sources are mounted
```

Comment:
Is $VAR_NAME is the only available format? Or `${VAR_NAME}` is available as well?

Reply:
`${VAR_NAME}` should work as well. But I guess this will depend on implementation.